validator/endpoints/tasks.py

```diff
@@ -218,76 +190,22 @@ async def get_leaderboard(
         logger.info(f"Trying node {node}")
         try:
             node_stats = await submissions_and_scoring_sql.get_all_node_stats(node.hotkey, config.psql_db)
-            leaderboard_rows.append(LeaderboardRow(
-                hotkey=node.hotkey, stats=node_stats))
+            leaderboard_rows.append(LeaderboardRow(hotkey=node.hotkey, stats=node_stats))
         except Exception as e:
-            logger.error(
-                f"Error processing scores for hotkey {node.hotkey}: {e}")
+            logger.error(f"Error processing scores for hotkey {node.hotkey}: {e}")
             continue
     return leaderboard_rows
 
 
 def factory_router() -> APIRouter:
-    router = APIRouter()
-
-    router.add_api_route(
-        "/v1/tasks/create",
-        create_task,
-        response_model=NewTaskResponse,
-        tags=["Training"],
-        methods=["POST"],
-        dependencies=[Depends(get_api_key)]
-    )
-
-    router.add_api_route(
-        "/v1/tasks/{task_id}",
-        get_task_status,
-        response_model=TaskStatusResponse,
-        tags=["Training"],
-        methods=["GET"],
-        dependencies=[Depends(get_api_key)]
-    )
-
-    router.add_api_route(
-        "/v1/tasks/delete/{task_id}",
-        delete_task,
-        response_model=NewTaskResponse,
-        tags=["Training"],
-        methods=["DELETE"],
-        dependencies=[Depends(get_api_key)]
-    )
+    router = APIRouter(tags=["Gradients On Demand"], dependencies=[Depends(get_api_key)])
 
-    router.add_api_route(
-        "/v1/tasks/task_results/{task_id}",
-        get_task_results,
-        response_model=TaskResultResponse,
-        tags=["Training"],
-        methods=["GET"],
-    )
-
-    router.add_api_route(
-        "/v1/tasks/node_results/{hotkey}",
-        get_node_results,
-        response_model=AllOfNodeResults,
-        tags=["Training"],
-        methods=["GET"],
-    )
+    router.add_api_route(TASKS_CREATE_ENDPOINT, create_task, methods=["POST"])
```

Comment:
Did wonder why we even have response_models when it's already explicit within the task